jetty-server/src/main/java/org/eclipse/jetty/server/session/AbstractSessionDataStore.java

```diff
@@ -39,8 +39,52 @@
     protected SessionContext _context; //context associated with this session data store
     protected int _gracePeriodSec = 60 * 60; //default of 1hr 
     protected long _lastExpiryCheckTime = 0; //last time in ms that getExpired was called
+    protected long _lastOrphanSweepTime = 0; //last time in ms that we deleted orphaned sessions
     protected int _savePeriodSec = 0; //time in sec between saves
-
+    
+    /**
+     * Small utility class to allow us to
+     * return a result and an Exception
+     * from invocation of Runnables.
+     *
+     * @param <V> the type of the result.
+     */
+    private class RunnableResult<V>
```

Comment:
`Result` is a better name, as it has nothing to do with a `Runnable`

Reply:
Done.